front/src/applications/stdcmV2/views/StdcmViewV2.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useRef, useEffect } from 'react';
+
+import { Button } from '@osrd-project/ui-core';
+import { Location, ArrowUp, ArrowDown } from '@osrd-project/ui-icons';
+import { useSelector } from 'react-redux';
+
+import STDCM_REQUEST_STATUS from 'applications/stdcm/consts';
+import useStdcm from 'applications/stdcm/views/useStdcm';
+import { useOsrdConfSelectors } from 'common/osrdContext';
+import ScenarioExplorer from 'modules/scenario/components/ScenarioExplorer';
+import { Map } from 'modules/trainschedule/components/ManageTrainSchedule';
+
+import StdcmConsist from '../components/StdcmConsist';
+import StdcmDefaultCard from '../components/StdcmDefaultCard';
+import StdcmDestination from '../components/StdcmDestination';
+import StdcmHeader from '../components/StdcmHeader';
+import StdcmLoader from '../components/StdcmLoader';
+import StdcmOrigin from '../components/StdcmOrigin';
+
+// TODO
+// - translate labels
+
+export default function StdcmViewV2() {
```

Comment:
```suggestion
const StdcmViewV2 = () => {
```
and at the end of the file:
```
export default StdcmViewV2;
```